Makefile

```diff
@@ -2,19 +2,27 @@ APP ?= bin/go-skeleton
 ENV ?= dev
 PORT ?= 8080
 
+MODULE = github.com/ivanterekh/go-skeleton
+VERSION = $(shell git tag)
```

Comment:
suggest to change to 
`$(git describe --tags $(git rev-list --tags --max-count=1))`